openpype/lib/transcoding.py

```diff
@@ -1037,3 +1037,56 @@ def convert_ffprobe_fps_to_float(value):
     if divisor == 0.0:
         return 0.0
     return dividend / divisor
+
+
+def convert_colorspace(
+    input_path,
+    out_filepath,
+    config_path,
+    source_colorspace,
+    target_colorspace,
+    logger=None
+):
+    """Convert source files from one color space to another.
+
+    Filenames of input files are kept so make sure that output directory
+    is not the same directory as input files have.
+    - This way it can handle gaps and can keep input filenames without handling
+        frame template
+
+    Args:
+        input_path (str): Paths that should be converted. It is expected that
+            contains single file or image sequence of samy type.
```

Comment:
Paths or path? Is it a filepath? a folder? What happens if there are many different files in that folder?

It also feels a bit odd that this is called `input_path` but the output path is called `out_filepath`

Reply:
Not part of this PR. Comes from; https://github.com/ynput/OpenPype/pull/4291